M2477: Missing connector: "and"

carolinacmoravia opened the pull request:
@BillWagner, 
It seems that it  should be a connector between them. Please, review the proposed file change and if you agree, could you help to merge it?
Thanks in advance.

carolinacmoravia committed:
M2477: Missing connector: "and" 

@wiwagn, 
It seems that it  should be a connector between them. Please, review the proposed file change and if you agree, could you help to merge it?
Thanks in advance.

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

BillWagner commented on docs/csharp/programming-guide/concepts/linq/introduction-to-linq-queries.md:
```diff
@@ -85,7 +85,7 @@ IQueryable<Customer> custQuery =
  The query specifies what information to retrieve from the data source or sources. Optionally, a query also specifies how that information should be sorted, grouped, and shaped before it is returned. A query is stored in a query variable and initialized with a query expression. To make it easier to write queries, C# has introduced new query syntax.  
   
- The query in the previous example returns all the even numbers from the integer array. The query expression contains three clauses: `from`, `where` and `select`. (If you are familiar with SQL, you will have noticed that the ordering of the clauses is reversed from the order in SQL.) The `from` clause specifies the data source, the `where` clause applies the filter, and the `select` clause specif ...
+ The query in the previous example returns all the even numbers from the integer array. The query expression contains three clauses: `from`, `where` and `select`. (If you are familiar with SQL, you will have noticed that the ordering of the clauses is reversed from the order in SQL.) The `from` clause specifies the data source, the `where` clause applies the filter, and the `select` clause specif ...
```
I followed both links, and the second one was migrated by mistake.
I'd prefer if you just removed it.

BillWagner reviewed (changes requested):
@carolinacmoravia Thank you for spotting this and submitting a fix. We appreciate it. 

that second link should be removed. Once you make that change, we'll merge this.

Thanks again.

carolinacmoravia committed:
Update introduction-to-linq-queries.md

Deleting content as indicated. 
@BillWagner. please, review and help to merge. Thank you.

BillWagner commented:
Thanks again @carolinacmoravia  I'll merge as soon as the builds finish.
